flixel/FlxCamera.hx

```diff
@@ -493,8 +493,11 @@ class FlxCamera extends FlxBasic
 	var _point:FlxPoint = FlxPoint.get();
 
 	/**
-	 * Internal, the filters array to be applied to the camera.
+	 * The filters array to be applied to the camera. Default value to null for performance reasons.
 	 */
+	public var filters:Array<BitmapFilter>;
+
+	@:deprecated("_filters is deprecated, use filters instead")
 	var _filters:Array<BitmapFilter>;
```

Comment:
add a getter and setter to _filters so it sets the new filters field, we need this for backwards compatibility for the people that were using _filters with @:privateAccess

Reply:
Bet